GUI/DeusEx/Classes/DXRMenuScreenNewGameExtended.uc

```diff
@@ -0,0 +1,141 @@
+class DXRMenuScreenNewGameExtended extends DXRMenuScreenNewGame;
+
+#exec TEXTURE IMPORT FILE="Textures\MenuNewGameBackgroundExtended_1.pcx"	NAME="MenuNewGameBackgroundExtended_1"	GROUP="DXRandoUI" MIPS=Off
```

Comment:
normally we put the texture imports in `DXRando\DeusEx\Classes\DXRandoImageTextures.uc`, but we actually do have a couple of strays